src/app/project/board/board.component.ts

```diff
@@ -28,31 +30,35 @@ export class BoardComponent implements OnInit {
     // project data
     project: ReadProject;
 
-    projectMembers: ReadUser[] = [];
-
-    // i18n setup
-    itemPluralMapping = {
-        member: {
-            '=1': '1 User',
-            other: '# Users'
-        },
-        ontology: {
-            '=1': '1 Ontology',
-            other: '# Ontologies'
-        },
-        keyword: {
-            '=1': '1 Keyword',
-            other: '# Keywords'
-        }
-    };
+    color: string = 'primary';
+
+    // variables to store metadata information
+    metadata: object;
+    datasetMetadata: object;
+    projectMetadata: object;
+    attribution: object;
```

Comment:
Primitive object types could be replaced by interfaces, but it can be also done in another iteration since the data model is kind of liquid ;)